spring-cloud-gcp-data-datastore/src/test/java/com/google/cloud/spring/data/datastore/core/mapping/DatastoreMappingContextTests.java

```diff
@@ -69,6 +70,14 @@ public void testGetInvalidEntity() {
 				.hasMessage("Unable to find a DatastorePersistentEntity for: class java.lang.Integer");
 	}
 
+	@Test
+	public void testGetSimpleTypeTimestamp() {
```

Comment:
`testTimestampNotAnEntity`?

Reply:
Since `simpleTypeHolder` is a private member, this was the closest I could get to test that `Timestamp` is treated as a simple type via `shouldCreatePersistentEntityFor` logic.
Let me change the test name and leave a short comment.